File: Colleen/Go/Colleen.go
package main

import "fmt"

var s = "package main%c%cimport %cfmt%c%c%cvar s = %c%s%c%c%cfunc other() {%c    fmt.Printf(s, 10, 10, 34, 34, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 9, 10, 9, 10, 9, 10, 9, 10, 10)%c}%c%c/*%cGlobal Scope Comment%c*/%cfunc main() {%c%c/*%c%c  Main Scope Comment%c%c*/%c%cother()%c}%c"

func other() {
    fmt.Printf(s, 10, 10, 34, 34, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 9, 10, 9, 10, 9, 10, 9, 10, 10)
}

/*
Global Scope Comment
*/
func main() {
	/*
	  Main Scope Comment
	*/
	other()
}

File: Colleen/Rust/Colleen.rs
#![allow(non_snake_case)]

/*
  Global Scope Comment
*/
fn other() {
    let a = "#![allow(non_snake_case)]\n\n/*\n  Global Scope Comment\n*/\nfn other() {\n    let a = ";
    let c = "\n    ";
    let b = "\n    print!(\"{}{:?};{}let c = {:?};{}let b = {:?};{}\", a, a, c, c, c, b, b)\n}\n\nfn main() {\n    /*\n      Main Scope Comment\n    */\n    other()\n}\n";
    print!("{}{:?};{}let c = {:?};{}let b = {:?};{}", a, a, c, c, c, b, b)
}

fn main() {
    /*
      Main Scope Comment
    */
    other()
}
